pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -160,7 +160,7 @@ func (c *Controller) reconcile(qualifiedName util.QualifiedName) util.Reconcilia
 
 	limitedScope := c.controllerConfig.TargetNamespace != metav1.NamespaceAll
 	if limitedScope && syncEnabled && !typeConfig.GetNamespaced() {
-		glog.Infof("Skipping start of sync & status controller for cluster-scoped resource %q.  It is not required for a namespaced federation control plane.", typeConfig.GetFederatedType().Kind)
+		glog.V(3).Infof("Skipping start of sync & status controller for cluster-scoped resource %q.  It is not required for a namespaced federation control plane.", typeConfig.GetFederatedType().Kind)
```

Comment:
Maybe this message should only be omitted once per kind?

Reply:
I make log level same as 
```
glog.V(3).Infof("Running reconcile FederatedTypeConfig for %q", key)
```
in front of reconcile.

So, when it tells concile, it shows skipping message to cluster-scoped resource. I think we do not need to make more logic for this message.